tests/integration/sugar/test_wallet.py

```diff
@@ -78,57 +109,61 @@ async def test_generate_faucet_wallet_rel_sub(self, client):
         )
         self.assertTrue(response.is_successful())
 
-    async def test_generate_faucet_wallet_testnet_async_websockets(self):
-        async with AsyncWebsocketClient(
-            "wss://s.altnet.rippletest.net:51233"
-        ) as client:
-            await generate_faucet_wallet_and_fund_again(self, client)
+    # Custom host tests
 
-    async def test_generate_faucet_wallet_custom_host_async_websockets(self):
+    async def _test_generate_faucet_wallet_custom_host_async_websockets(self):
```

Comment:
Instead of naming these `_test_...` - can we name them something like `_parallel_test` then search for `_parallel` in the above code that's currently looking for `_test_`? 
(I would not guess from just reading the test name that these are parallel, especially since having an underscore in front is already meaningful in a different way for "private function".